modules/plugins/completion/nvim-cmp/nvim-cmp.nix

```diff
@@ -7,6 +7,8 @@ in {
     autocomplete = {
       enable = mkEnableOption "enable autocomplete" // {default = false;};
 
+      alwaysComplete = mkEnableOption "Automatically show completion." // {default = true;};
```

Comment:
`mkEnableOption` automatically prefixes the description with "Whether to enable". This should be `mkEnableOption "automatically showing completion menu."` so that the documentation is grammatically correct.

While you're on that, maybe remove the "enable" prefix from the option definition above.